### DISCUSSION TITLE
Copilot for JetBrains does not support 2024.1 EAP

### DISCUSSION THREAD

**Discussion by pan93412** [community/community#88790] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Copilot for JetBrains does not support JetBrains 2024.1 EAP (build 241.8102.131).

![CleanShot 2024-01-19 at 11 16 12@2x](https://github.com/community/community/assets/28441561/b19b6c32-737b-432e-95cc-1ff5907cd27b)

**Comment by baodden** [comment]
Supporting, or at least not restricting use from EAP, would be nice. The new Webstorm Next/EAP has some improvements I want to start using right away.

**Comment by pan93412** [comment, marked as answer]
[GitHub Copilot 1.4.12.4386](https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/472811) has supported 2024.1 EAP.

**Comment by Scott-G** [comment]
The 2024.2 EAP has been out for almost 2 weeks and STILL the CoPliot plugin does not support it.

Are we going to have this every time a new EAP comes out, now?

**Reply by pan93412** [reply]
It may be better to open a new thread for this. Their staff may not check if there is any new question in the “answered” discussion.

**Reply by rhydianroberts** [reply]
The latest nightly now supports 2024.2: https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/nightly/555078
